platform/mbed_power_mgmt.h

```diff
@@ -205,6 +205,32 @@ static inline void system_reset(void)
 {
     NVIC_SystemReset();
 }
+ 
+/** Provides the time spent in sleep mode, since system is up and running
```

Comment:
`since system is up and running` that's a bit confusing, can just say from boot? Also we should call out in each of the functions (`@note`) it only works if platform supports LP ticker.